### PR TITLE
added: list component added, variables updated

### PR DESCRIPTION
**Pull request by rishbitsnbytes**
Added List Component.

Added following types of lists : 

Spaced List
Inline List
Stacked List
Stacked Reversed List

lists

### CODE REVIEW

File: Components/Lists/lists.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        <li>List item with numbers</li>
+        <li>List item with numbers</li>
+        <li>List item with numbers</li>
+        <li>List item with numbers</li>
+    </ol>
+    <br>
+    
+    <!-- Reverse Stacked List -->
+
+    <ol class="list-stacked" reversed>
```
Comment (vidhi-thakur) [inline Components/Lists/lists.html:58]: Just a suggestion: It looks a little confusing here. The comment says `Reverse Stacked List` but the class name is `list-stacked`. Can update the class name here.

**Review by vidhi-thakur — APPROVED** [review]
good to merge

**Review by ravikumar0403 — APPROVED** [review]
Looks good to merge

**Review by TejasShekar — APPROVED** [review]
Looks good to merge.

File: Components/Lists/lists.html (hunk 1 above)
Comment (rishbitsnbytes) [reply Components/Lists/lists.html:58]: Sure, thanks for pointing it out.

**Pull request merged**